lib/services/extensions-service-base.ts

```diff
@@ -52,7 +52,7 @@ export class ExtensionsServiceBase {
 
 			if( this.shouldUpdatePackage(cachedVersion, extensionVersion) ) {
 				this.$logger.printInfoMessageOnSameLine(`Updating ${packageName} package...`);
-				let zipFileName = temp.path({ path:  path.join(this.cacheDir, packageName + ".zip") });
+				let zipFileName = path.join(this.cacheDir, packageName + ".zip");
```

Comment:
The temp module have never worked. We used wrong syntax and it created a temporary file which we never used.